src/config/local_app_dependency_test.go

```diff
@@ -109,19 +108,43 @@ var _ = Describe("LocalAppDependency", func() {
 			})
 		})
 
+		var _ = Describe("GetDeploymentVariables", func() {
+			It("should return the correct deployment config for exocom", func() {
+				variables, err := exocomProd.GetDeploymentVariables()
+				Expect(err).NotTo(HaveOccurred())
+				serviceData, err := json.Marshal(map[string]map[string]interface{}{
+					"api-service":      {},
+					"external-service": {},
+					"html-server": {
+						"receives": []interface{}{"todo.created"},
+						"sends":    []interface{}{"todo.create"},
+					},
+					"todo-service": {
+						"receives": []interface{}{"todo.create"},
+						"sends":    []interface{}{"todo.created"},
+					},
+					"users-service": {
+						"receives": []interface{}{"mongo.list", "mongo.create"},
+						"sends":    []interface{}{"mongo.listed", "mongo.created"},
+						"translations": []interface{}{
+							map[string]interface{}{
+								"internal": "mongo create",
+								"public":   "users create",
+							},
+						},
+					},
+				})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(variables).To(Equal(map[string]string{
+					"SERVICE_DATA": string(serviceData),
+				}))
+			})
+		})
+
 		var _ = Describe("GetDeploymentConfig", func() {
 			It("should return the correct deployment config for exocom", func() {
 				actual, err := exocomProd.GetDeploymentConfig()
 				Expect(err).NotTo(HaveOccurred())
-				expectedServiceRoutes := []string{
-					`{"receives":["users.listed","users.created"],"role":"external-service","sends":["users.list","users.create"]}`,
-					`{"receives":["todo.create"],"role":"todo-service","sends":["todo.created"]}`,
-					`{"namespace":"mongo","receives":["mongo.list","mongo.create"],"role":"users-service","sends":["mongo.listed","mongo.created"]}`,
-					`{"receives":["todo.created"],"role":"html-server","sends":["todo.create"]}`,
-				}
-				for _, serviceRoute := range expectedServiceRoutes {
-					Expect(strings.Contains(actual["serviceRoutes"], serviceRoute))
```

Comment:
This wasn't actually testing anyting. It needed the `.To()` in order to test. Note with tests: always try to see them fail when writing / refactoring.